frontend/packages/integration-tests-cypress/tests/i18n/pseudolocalization.spec.ts

```diff
@@ -42,4 +44,33 @@ describe('Localization', () => {
     cy.visit('/dashboards?pseudolocalization=true&lng=en');
     cy.byTestID('utilization-card-item-text').isPseudoLocalized();
   });
+
+  it('pseudolocalizes monitoring pages', () => {
+    const monitoringPages = ['alerts', 'silences', 'alertrules'];
+    monitoringPages.forEach((page) => {
+      cy.visit(`/monitoring/${page}?pseudolocalization=true&lng=en`);
+      if (page === 'alerts' || page === 'alertrules') {
+        listPage.rows.shouldBeLoaded();
+      }
+      // check applied filters are i18ned then clear all filters
+      cy.get('.pf-c-chip-group__main').then((groups) => {
+        if (groups.length >= 1) {
+          cy.get('[class*="pf-c-chip-group"]').isPseudoLocalized();
+        }
+        listPage.filter.clearAllFilters();
+      });
+      cy.get(ListPageSelector.tableColumnHeaders).isPseudoLocalized();
+      listPage.filter.clickFilterDropdown();
+      cy.get('.co-filter-dropdown-item__name').isPseudoLocalized();
+      if (page === 'alerts' || page === 'alertrules') {
+        if (page === 'alerts') {
+          cy.get(DetailsPageSelector.horizontalNavTabs).isPseudoLocalized();
+        }
+        listPage.filter.clickSearchByDropdown();
+        cy.get('[data-test-id="dropdown-menu"]').isPseudoLocalized();
+      } else {
+        cy.get('.pf-c-button.pf-m-primary').isPseudoLocalized();
```

Comment:
I would add a `data-test='create-silence-btn'` attribute to the [Create Silence](https://github.com/openshift/console/blob/973141d456fb5f2cd7e6b3f1d66ad79bdcf8ac73/frontend/public/components/monitoring/alerting.tsx#L1568) button then use `cy.byTestID('create-silence-btn')`

Reply:
that looks much better, let me try to update